include/deal.II/grid/grid_tools.h

```diff
@@ -1079,6 +1079,48 @@ namespace GridTools
     std::map<typename Triangulation<Container::dimension,Container::space_dimension>::active_cell_iterator,
     typename Container::active_cell_iterator> &patch_to_global_tria_map);
 
+  /**
+   * This function runs through the degrees of freedom defined by the
+   * Container and for each dof constructs a vector of active_cell_iterators
+   * representing the cells of support of the nodal basis element
+   * at that degree of freedom. This function was designed for the
+   * implementation of local projections, for instance the Clement interpolant,
+   * in conjunction with other local patch functions like
+   * GridTools::build_triangulation_from_patch.
+   *
+   * Containers built on top of Triangulation<dim> or
+   * parallel:distributed::Triangulation<dim> are supported and handled
+   * appropriately.
+   *
+   * It is necessary that the finite element underlying the MeshType used has
+   * degrees of freedom that are logically associated to a vertex, line, quad,
+   * or hex.  This includes both nodal degrees of freedom and other modal types
+   * of dofs that are associated to an edge, etc.  The result is the patch of
```

Comment:
I think that what you say in lines 1095-1098 here covers any conceivable element that could be implemented in deal.II. So I would just omit these two sentences.